packages/routes-gen-solid-start/README.md

```diff
@@ -0,0 +1,17 @@
+# @routes-gen/solid-start
+
+This package is a [**routes-gen**](https://github.com/sandulat/routes-gen) driver for [SolidStart](https://github.com/solidjs/solid-start).
+
+## Installation
+
+```
+yarn add routes-gen @routes-gen/solid-start
+```
+
+## Usage
+
+```
+yarn routes-gen -d @routes-gen/solud-start
```

Comment:
yarn routes-gen -d @routes-gen/`solid`-start
Should be `solid` instead of `solud`